Documentation/ExtensionArchitecture/FileStructure/Configuration/Sets/Index.rst

```diff
@@ -1,63 +1,75 @@
-.. include:: /Includes.rst.txt
-.. _extension-configuration-sets:
+:navigation-title: Sets
 
-============
-:file:`Sets`
-============
+..  include:: /Includes.rst.txt
+..  _extension-configuration-sets:
+
+=====================================
+Extension folder `Configuration/Sets`
+=====================================
 
 ..  versionadded:: 13.1
     `Site sets <https://docs.typo3.org/permalink/t3coreapi:site-sets>`_ have
-    been introduces.
+    been introduced.
 
 In this directory TYPO3 extensions can provide their
 `Site sets <https://docs.typo3.org/permalink/t3coreapi:site-sets>`_.
 
 Each set must be saved in its own directory and have at least a file called
 :file:`config.yaml`.
 
-..  contents:: Files in a set:
-
 .. _extension-configuration-sets-config-yaml:
 
-:file:`config.yaml` (mandatory)
-===============================
+..  typo3:file:: config.yaml
+    :name: set-config-yaml
+    :scope: set
+    :regex: /^.*Configuration\/Sets\/[\w\-]+\/config\.yaml$/
+    :shortDescription: Contains the definition of a site set. Mandatory.
 
-Contains the `definition of a site set <https://docs.typo3.org/permalink/t3coreapi:site-sets-definition>`_
-and its dependencies.
+    Contains the `definition of a site set <https://docs.typo3.org/permalink/t3coreapi:site-sets-definition>`_
+    and its dependencies.
 
 Example:
 
 ..  include:: /ApiOverview/SiteHandling/_Sets/_site-package/_config.rst.txt
 
 .. _extension-configuration-sets-settings-yaml:
 
-:file:`settings.yaml`
-=====================
+..  typo3:file:: settings.yaml
+    :name: set-settings-yaml
+    :scope: set
+    :regex: /^.*Configuration\/Sets\/[\w\-]+\/settings\.yaml$/
+    :shortDescription: Override settings defined by other sets.
 
-In this file an extension can override settings defined by other sets. For
-example :ref:`Settings provided by site set "Fluid Styled Content" <typo3/cms-fluid-styled-content:site-set-fluid-styled-content-settings>`:
+    In this file an extension can override settings defined by other sets. For
+    example :ref:`Settings provided by site set "Fluid Styled Content" <typo3/cms-fluid-styled-content:site-set-fluid-styled-content-settings>`:
 
 ..  include:: /ApiOverview/SiteHandling/_Sets/_site-package/_settings.rst.txt
 
 .. _extension-configuration-sets-settings-definitions-yaml:
 
-:file:`settings.definitions.yaml`
-=================================
+..  typo3:file:: settings.definitions.yaml
+    :name: set-settings-definitions-yaml
+    :scope: set
+    :regex: /^.*Configuration\/Sets\/[\w\-]+\/settings\.definitions\.yaml$/
+    :shortDescription: Contains the setting definition and defaults of a set.
```

Comment:
`of a set`? Sorry. Can't provide a suggestion here as I don't know what you want to write here

Reply:
`defaults of a set` sound wrong while reading, but I just have checked it with deepl write and other tools, but it seems to be correct. Sorry, for interrupt. I start merging.